start-local.sh

```diff
@@ -7,7 +7,7 @@ if [ ! -d "$LOGS" ]; then
     exit 1
 fi
 
-if [ `pwd | sed s/.*src//` != "/github.com/lacker/coinkit" ]; then
+if [[ `pwd | sed s/.*src//` != *coinkit ]]; then
```

Comment:
Does it actually have to be in /github.com/lacker/coinkit after all?

Reply:
I think **technically** the working directory doesn't have to be that, but it does have to exist.